docs/source/examples/cnn.rst

```diff
@@ -99,29 +95,31 @@ prediction result by calculating :meth:`~chainer.functions.softmax`.
   training and other modes, it was common that it held an attribute
   that represented its running mode or was provided with the mode
   from outside as an argument. In Chainer v2, it is recommended to use
-  the global configuration `chainer.config.train` to switch the running mode.
+  the global configuration ``chainer.config.train`` to switch the running mode.
 
 If you don't want to write ``conv1`` and the other layers more than once, you
-can also write the model like in this way:
+can also write the same model like in this way:
 
 .. testcode::
 
+    from functools import partial
+
     class LeNet5(Chain):
         def __init__(self):
             super(LeNet5, self).__init__()
             net = [('conv1', L.Convolution2D(1, 6, 5, 1))]
-            net += [('_sigm1', F.Sigmoid())]
-            net += [('_mpool1', F.MaxPooling2D(2, 2))]
+            net += [('_sigm1', F.sigmoid)]
```

Comment:
Could you let me know why you avoid using instances of FunctionNode themselves?

Reply:
Because FunctionNodes are not callable (unlike old-style Functions), and FunctionNodes are not public API for users.

Note: this example is a user-implemented version of `Sequential` added in v4.
https://docs.chainer.org/en/stable/reference/generated/chainer.Sequential.html
This example should be rewritten to use `Sequential` once `Sequential` feature becomes non-experimental.

Reply:
Thank you for explanation. I got it.